packages/experimental/RadioGroup/src/__tests__/__snapshots__/RadioGroupExperimental.test.tsx.snap

```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
This doesn't seem related to your change, maybe it got added when you ran yarn test?

Reply:
Agreed, I thought I removed this when I saw it was part of my CL, thanks for catching this.